fe/src/project/AspNetCore/areas/Orders/components/NewOrderForm/NewOrderForm.js

```diff
@@ -557,11 +600,18 @@ NewOrderForm.propTypes = {
     ordersUrl: PropTypes.string.isRequired,
     placeOrderUrl: PropTypes.string.isRequired,
     navigateToOrdersListText: PropTypes.string.isRequired,
-    expectedDeliveryabel: PropTypes.string.isRequired,
+    expectedDeliveryLabel: PropTypes.string.isRequired,
     uploadOrderFileUrl: PropTypes.string.isRequired,
     orLabel: PropTypes.string.isRequired,
     dropOrSelectFilesLabel: PropTypes.string.isRequired,
-    dropFilesLabel: PropTypes.string.isRequired
+    dropFilesLabel: PropTypes.string.isRequired,
+    initCustomOrderLabel: PropTypes.string.isRequired,
+    customOrderLabel: PropTypes.string.isRequired
 };
 
 export default NewOrderForm;
+
```

Comment:
**suggestion:** let's always remove comments like this from code. We have source control for it.

Reply:
Sorry, i didn't notice it